app.py

```diff
@@ -452,6 +454,25 @@ def get_first_name(schedule):
     return schedule.get("preferred_name") or schedule["firstname"]
 
 
+def get_github_info():
+    # this uses PyGithub module
+    # using an access token from a person who can access epschedule
+    g = gh(os.environ["GH_TOKEN"])
+    repo = g.get_repo("EastsidePreparatorySchool/epschedule")
+    # get arr of commits
+    commitsArr = repo.get_commits()
+    # print info about last 3
+    result = []
+    for repo_num in range(0, 3):
+        commit_name = str(commitsArr[repo_num].commit.message).split("\n")[0]
+        commit_author = str(commitsArr[repo_num].commit.author)[16:-2]
```

Comment:
add comments explaining what you're doing here. Also, aren't these values strings already (and don't need str())?

Reply:
they are a commit object and it's hard to pass without making them strings

Reply:
isn't `.message` a string?
https://pygithub.readthedocs.io/en/latest/github_objects/GitCommit.html#github.GitCommit.GitCommit

Reply:
I think you also probably want `author.name` rather than parsing the stringized version